docs/spec_draft.md

```diff
@@ -343,6 +344,67 @@ logical operation.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.batch_norm_inference
+
+### Semantics
+
+Normalizes the `operand` tensor across batch and spatial dimensions, using
+`mean` and `variance` provided in input, for each feature in the `feature_index`
```

Comment:
can we skip `provided in input`. That is because it seems we are qualifying some as input and others (`feature_index`) not.